README.md

```diff
@@ -129,9 +126,55 @@ If this process sounds familiar, it's because it's basically the same as having
 
 - **JQuery** Anytime you see that money sign, you're working with a Jquery object. That Jquery object is set by referring to a CSS selector for example $('body') to select the body element of the DOM.
 
+### Questions from Homework
+**What parts of the page does the ApplicationView directly manage?**
+
+The ApplicationView manages the "New Contact" form.
```

Comment:
Should we mention that the ApplicationView also directly manages the RolodexView?

I think we should have had some questions that asked about that directly, but and important concept we want to get across is how Views often create and manage other child Views, in a way that mirrors the hierarchy of DOM elements that they control.